data/mods/Aftershock/README.md

```diff
@@ -1,9 +1,10 @@
+* As it becomes a total conversion mod it will look very different from the base game of Cataclysm: Dark Days Ahead and may not interact kindly with other mods.  If you've been away for a while you may wonder where some parts of Aftershock have gone.  In the process of becoming a total conversion mod we are eliminating items that no longer fit the theme.  For example, all the Mad Max aesthetics are currently removed.  For them to come back would require someone to write up a location/faction that made them make sense in a futuristic extrasolar colony.  
+Aftershock is being developed into a full conversion mod set in a mostly abandoned extrasolar colony named Salus 4.  The PC has arrived on Salus for various reasons depending on their profession.  Eventually we hope to have multiple potential end game situations such as finding what you came here for and leaving the planet with enough money to retire to more secret endings.  There will no longer be zombies but instead a variety of alien and human based enemies.  Toe to toe combat will likely be more deadly in Aftershock with more emphasis on picking your battles and only fighting when absolutely necessary.  Special locations may have a different combat aesthetic for example you may enter a dungeon like zone with hordes of small weak enemies that can't be avoided.  As part of this concept where different situations require different solutions we have implemented bionic slots but made the process of changing out CBMs much easier with less chance of failure.  This is to encourage players to change their loadout based on where they are going and what they are doing.
 
 ### Areas seeking contributors
-Faction interactions and new factions.  Currently there are two new factions in Aftershock. PrepNet and Whately Clan.  PrepNet is at war with the Mycus and the Whately's are mad scientists.  
-Uplifted animal mobs and mutation lines are always wanted.  I'd love to see someone make a Rabbit or Jackelope line.  
+Faction interactions and new factions.  Currently there are two new factions in Aftershock. PrepNet and Whately Clan.  PrepNet is seeking to build an independent colony and the Whately's are mad scientists.  
+Uplifted animal mobs and mutation lines are always wanted.  I'd love to see someone make a Rabbit or Jackelope line.  Currently we have Mi-Go, Mastodon, and Cecaelian mutation trees.
```

Comment:
Any differences on the mutation background (as in, determining what's more/less likely)?

Reply:
Mutations are no longer Blob driven in Aftershock, so there will be things more often seen in literature that aren't likely in Cataclysm: DDA such as animals that are similar in size to their current versions but have some ability to speak or other increased abilities.

Reply:
Hmm... what's the climate like on this colony? (That may mean needed changes in JSONs currently set for New England, BTW.) If cold - Yeti?

Reply:
It's definitely colder than New England and so that's one set of the work is to make those kinds of changes.

Reply:
Its very arctic/cold, very cold temperature ranges that make living without adequate protection impossible

Reply:
Item, possibly with a bionic version/upgrade - ice skates. (The bionic part could be anything from blades that come out of your bionic feet to helping you balance on the skates.)